runtime/src/weights/pallet_xcm_benchmarks.rs

```diff
@@ -0,0 +1,308 @@
+    }
+    fn expect_asset(assets: &Assets) -> sp_weights::Weight {
+        assets.weigh_multi_assets(XcmGenericWeight::<Runtime>::expect_asset())
+    }
+    fn expect_origin(_: &core::option::Option<Location>) -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::expect_origin()
+    }
+    fn expect_error(_: &core::option::Option<(u32, Error)>) -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::expect_error()
+    }
+    fn expect_transact_status(_: &MaybeErrorCode) -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::expect_transact_status()
+    }
+    fn query_pallet(_: &Vec<u8>, _: &QueryResponseInfo) -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::query_pallet()
+    }
+    fn expect_pallet(_: &u32, _: &Vec<u8>, _: &Vec<u8>, _: &u32, _: &u32) -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::expect_pallet()
+    }
+    fn report_transact_status(_: &QueryResponseInfo) -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::report_transact_status()
+    }
+    fn clear_transact_status() -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::clear_transact_status()
+    }
+    fn universal_origin(_: &Junction) -> sp_weights::Weight {
+        // ZKV does not currently support universal origin operations
+        Weight::MAX
+    }
+    fn export_message(_: &NetworkId, _: &Junctions, _: &Xcm<()>) -> sp_weights::Weight {
+        // ZKV does not currently support asset locking operations
+        Weight::MAX
+    }
+    fn lock_asset(_: &Asset, _: &Location) -> sp_weights::Weight {
+        // ZKV does not currently support asset locking operations
+        Weight::MAX
+    }
+    fn unlock_asset(_: &Asset, _: &Location) -> sp_weights::Weight {
+        // ZKV does not currently support asset locking operations
+        Weight::MAX
+    }
+    fn note_unlockable(_: &Asset, _: &Location) -> sp_weights::Weight {
+        // ZKV does not currently support asset locking operations
+        Weight::MAX
+    }
+    fn request_unlock(_: &Asset, _: &Location) -> sp_weights::Weight {
+        // ZKV does not currently support asset locking operations
+        Weight::MAX
+    }
+    fn set_fees_mode(_: &bool) -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::set_fees_mode()
+    }
+    fn set_topic(_: &[u8; 32]) -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::set_topic()
+    }
+    fn clear_topic() -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::clear_topic()
+    }
+    fn alias_origin(_: &Location) -> sp_weights::Weight {
+        // XCM Executor does not currently support alias origin operations
+        Weight::MAX
+    }
+    fn unpaid_execution(_: &WeightLimit, _: &core::option::Option<Location>) -> sp_weights::Weight {
+        XcmGenericWeight::<Runtime>::unpaid_execution()
+    }
+    fn pay_fees(_asset: &Asset) -> Weight {
+        unimplemented!("This should probably be implemented!")
```

Comment:
Just a reminder not to forget to update this, now that we have an implementation